### PR TITLE
Add support for using docker to run protonvpn

### PR DESCRIPTION
**Pull request by adambregenzer**

### CODE REVIEW

**Comment by adambregenzer** [comment]
start the container:
```
docker-compose linux-cli
```

```
route -n && protonvpn c -f && route -n && protonvpn d && route -n
```

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM python:3.5
```
Comment (morrissimo) [inline Dockerfile:1]: why python3.5? stale ubuntu package repo listings?

Comment (adambregenzer) [reply Dockerfile:1]: That's what they want, heh - https://github.com/ProtonVPN/linux-cli/blob/master/CONTRIBUTING.md

Comment (morrissimo) [reply Dockerfile:1]: > 2. Ensure that your code is compatible with the lowest Python Version supported by this project (Currently 3.5)

ha - so it shall be

**Review by morrissimo — APPROVED** [review]

**Pull request merged**